src/components/stretched-link-card/StretchedLinkCard.vue

```diff
@@ -139,6 +139,15 @@
                 <slot name="stretched-card-content" />
             </div>
 
+            <div class="vs-stretched-link-card__badges">
+                <VsBadge
```

Comment:
I think this would be better off as a slot instead of passing in badge labels as an array.

Also this would need unit tests for new props or slots.

Reply:
@GemmaLouise  good point. I've replaced the prop with a slot and added a new test for the slot